Data/Base/Shaders/Pipeline/DownscaleDepth.ezShader

```diff
@@ -51,7 +50,7 @@ float4 main(PS_IN Input) : SV_Target
   for (int i = 0; i < 4; ++i)
   {
     float2 samplePos = (screenPosition.xy + offsets[i]) * PixelSize;
-    depth = max(depth, DepthTexture.SampleLevel(DepthSampler, float3(samplePos, s_ActiveCameraEyeIndex), 0).r);
```

Comment:
Also needs a platform specific solution.

Reply:
Removed from this PR.

Reply:
I've added a platform function `ezSampleLevel_PointClampBorder` which abstracts this as well as a unit test.